README.md

```diff
@@ -12,100 +12,81 @@
 
 # Data Load Tool (DLT)
 
-Data Load Tool (DLT) enables simple, python-native data pipelining for data professionals.
+## What is DLT?
 
-It is an open source, scalable data loading framework that does not require any help from DevOps.
+Data Load Tool (DLT) is an open source python library for building data pipelines.
 
-Learn more with the **[Quickstart Guide](QUICKSTART.md)** and check out **[Example Sources](examples/README.md)** to get started.
+The goal of DLT is to make it easier and faster to build low-maintenance data pipelines.
 
-## How does it work?
+It's designed to run anywhere (Python 3.8+), so it can fit into your existing workflows or be scheduled independently (e.g. using GitHub Actions)
 
-DLT aims to simplify data loading for everyone.
+Learn more with the **[Quickstart Guide](QUICKSTART.md)** and check out **[Example Sources](examples/README.md)** to get started.
```

Comment:
I would put ahead a qualifier in which stage we are in with DLT ahead of  "Learn more with the **[Quickstart Guide](QUICKSTART.md)** and check out **[Example Sources](examples/README.md)** to get started." eg "While we have been using DLT in production for a while, this public version is in friendly viewing". I would want @TyDunn / @adrianbr to come up with a phrasing they like